docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -59,7 +59,7 @@ The workflow looks like this:
 
 ## Running tests
 
-If you haven't built the tests yet, run `src\tests\build[.cmd|.sh] nativeaot [Debug|Release] tree nativeaot`. This will build the smoke tests only - they usually suffice to ensure the runtime and compiler is in a workable shape. To build all Pri-0 tests, drop the `tree nativeaot` parameter. The `Debug`/`Release` parameter should match the build configuration you used to build the runtime.
+If you haven't built the tests yet, run `src\tests\build[.cmd|.sh] -nativeaot [Debug|Release] -tree:nativeaot`. This will build the smoke tests only - they usually suffice to ensure the runtime and compiler is in a workable shape. To build all Pri-0 tests, drop the `-tree:nativeaot` parameter. The `Debug`/`Release` parameter should match the build configuration you used to build the runtime.
```

Comment:
Command `src/tests/build.cmd nativeaot [Debug|Release] tree nativeaot` is working on Windows. There difference between Linux and Windows commads right now. For both building tests and for runing tests

Reply:
```suggestion
If you haven't built the tests yet, run `src\tests\build.cmd nativeaot [Debug|Release] tree nativeaot`, for Linux use `src/tests/build.sh -nativeaot [Debug|Release] -tree:nativeaot`. This will build the smoke tests only - they usually suffice to ensure the runtime and compiler is in a workable shape. To build all Pri-0 tests, drop the `tree nativeaot` parameter. The `Debug`/`Release` parameter should match the build configuration you used to build the runtime.
```

Reply:
that should be correct now.